css/index.css

```diff
@@ -1,3 +1,412 @@
+        width: 100%;
+        max-width: 800;
+        min-width: 600px;
+        margin: 0 auto;
+    }
+
+    .container {
+        margin-left: 20px;
+        padding: 0 0 0 0;
+        width: 95%;
+        display: flex;
+        flex-wrap: wrap;
+    }
+    
+    .container .main-nav {
+        display: flex;
+        justify-content: center;
+        align-items: column;
+        flex-wrap: wrap-reverse;
+        margin: 18px auto;
+        width: 100%;
+        height: 96px;
+}
+
+.container .main-nav img {
+    margin: -5px 0px 0 56px
+}
+
+.container .main-nav nav {
+    display: flex;
+    justify-content: space-around;
+    flex-flow: row wrap;
+    margin: ;
+}
+
+.container .main-nav nav a {
+    margin: 0px 18px 0px 48px
+}
+
+.container .top-content {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 64px 0px 0px 0px;
+    width: 756px;
+
+}
+
+.container .top-content .bold-top {
+    letter-spacing: .2em;
+    margin:4px 0px 51px 10px;
+}
+
+.container .top-content img {
+    display: none;
+    
+}
+
+.container .top-content .top-info {
+    margin: 0px 22px 0px 4px;
+}
+
+.container .top-content .top-info a {
+    text-align: center;
+    text-decoration: none;
+    color: black;
+    padding: 6px 0px 13px 7px;
+    margin: -13px 0px 75px 72px;
+    border: 1px solid black;
+    width: 168px;
+    height: 32px;
+    letter-spacing: .1em;
+}
+
+ /* Middle Content */
+
+ .container .mid-content {
+    display: flex; 
+    width: 100%;
+    padding: 0px 0px 0px 0px;
+    margin: 3px 0px 0px 20px;
+}
+
+.container .mid-content div {
+    padding: 0px 0px 0px 0px;
+    margin: 33px 44px 31px -10px;
+
+}
+
+.container .mid-ban .mid-img {
+    height: 152px;
+    /* width: 759px; */
+    padding: 0px 0px 0px 0;
+    margin: 0px 0px 36px 0px;
+    width: 100%;
+}
+
+
+  /* Bottom Content */
+
+  .container .mid-bottom {
+    display: flex;
+    justify-content: space-between;
+    width: 756px;
+    border-bottom: 2px solid black;
+    padding: 0px 87px 0px 0px;
+}
+
+.container .mid-bottom div {
+    padding: 0px 0px 0px 0px;
+    margin: 6px 6px 5px 10px;
+    width: 145px;
+}
+
+.container .mid-bottom div h1 {
+    margin: 25px 0px 0px 0px;
+}
+
+.container .mid-bottom div p {
+    padding: 0px 0px 0px 0px;
+    height: 127px;
+    width: 196px;
+    margin: 18px 0px 44px 0px;
+}
+
+}
+
+
+
+/* Mobile */
+@media (max-width: 500px) {
+    
+html, body {
+    width: 100%;
+    max-width:500;
+    min-width: 300px;
+    margin: 0 auto;
+}
+
+    .container {
+    }
+
+    .container .main-nav {
+       height: 877px;
+       justify-content: center;
+       align-items: center;
+       
+    }
+
+    .container .main-nav nav{
+        display: flex;
+        align-items: center;
+        flex-flow: column-reverse wrap;
+        margin: 50px;
+        justify-content: center;
+        
+    }
+
+    .container .main-nav nav a {
+        width: 100%;
+        border: 1px solid black;
```

Comment:
as a tweak, you have a double-border problem, which makes the internal borders thicker than the outlying ones. A solution could be to only set border-top on the anchor tags, and then set border-bottom (but not border-top) on the nav component